Feature/vplay 10641

balasaraswathy-n opened the pull request:
Rate correction issue is fixed

Commit:
VPLAY-10641: LLD Playback rate issue

Reason for change: LLD
Signed-off-by: Balasaraswathy_N <Balasaraswathy_N@comcast.com>
Priority: P0
Test Procedure: Validate LLD
Risks: None

balasaraswathy-n committed:
Merge pull request #446 from rdkcentral/dev_sprint_25_2

Rebasing

rdkcmf-jenkins commented on middleware/vendor/default/DefaultSocInterface.cpp:
```diff
@@ -267,4 +267,53 @@ bool DefaultSocInterface::IsVideoMaster(GstElement *videoSink, bool isRialto)
+		return false;
+    #endif
+
+	if(!pipeline)
```
<!-- Comment managed by coverity-report-output-v7 action, do not modify!
423fd0ad4fc0c739383164db021846bc
NOT_PRESENT
-->

Coverity issue no longer present as of: undefined
<details>
<summary>Show issue</summary>


# Coverity Issue - Structurally dead code
This code cannot be reached: "if (!pipeline) {
  do  {
  ...".

_Medium Impact, CWE-561_
_UNREACHABLE_



</details>

balasaraswathy-n committed:
Merge pull request #448 from rdkcentral/dev_sprint_25_2

Rebasing

balasaraswathy-n committed:
VPLAY-10641: LLD Playback rate issue

Reason for change: Resolving L1 failure
Signed-off-by: Balasaraswathy_N <Balasaraswathy_N@comcast.com>
Priority: P1
Test Procedure: Validate LLD
Risks: None

balasaraswathy-n committed:
Merge pull request #456 from rdkcentral/dev_sprint_25_2

 Rebasing

balasaraswathy-n committed:
Update DefaultSocInterface.cpp

pstroffolino committed:
Merge branch 'dev_sprint_25_2' into feature/VPLAY-10641_1
